### DISCUSSION TITLE
A glitch experienced during taking photo of document

### DISCUSSION THREAD

**Discussion by nixklai** [community/community#64431] [GitHub Education]
### Hi everyone,

Hi, I was trying to upload my salary statement from my University using my iPhone 12 but only the front-facing camera is working, while the rest of back cameras do not even show a black screen.

I awkwardly took a photo with my phone’s front camera, and it failed.

Do anyone experience this before? ~~I may have to think about using GitLab now~~

### A suggestion...(optional)

GitHub fixing the bug in the verification process.

**Comment by GhostOf0days** [comment]
A web app cannot control your phone's cameras not working, especially in the standardized way web apps allow you to upload photos. Switching services won't help you. Make sure you have your phone's software up-to-date, etc.

**Reply by GhostOf0days** [reply]
There are ways web apps can control your camera, but this is not something Github would have control over.
